go/vt/vtgate/planbuilder/select.go

```diff
@@ -271,7 +266,7 @@ func copyCommentsAndLocks(statement sqlparser.SelectStatement, sel *sqlparser.Se
 	query.Lock = sel.Lock
 	if sel.Into != nil {
 		if opcode != engine.Unsharded {
-			return vterrors.Errorf(vtrpcpb.Code_UNIMPLEMENTED, "INTO is not supported on sharded keyspace")
+			return vterrors.VT12001("INTO is not supported on sharded keyspace")
```

Comment:
```suggestion
			return vterrors.VT12001("INTO on sharded keyspace")
```

Reply:
resolved in [f2351bb](https://github.com/vitessio/vitess/pull/10738/commits/f2351bba65f347a8e5ce72ff87757ccaf20115af)